charts/eks/templates/metrics-proxy.yaml

```diff
@@ -0,0 +1,59 @@
+{{- if or .Values.proxy.metricsServer.nodes.enabled .Values.proxy.metricsServer.pods.enabled }}
```

Comment:
I think I would prefer to move this out of the helm chart and into the code here